docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -244,6 +244,23 @@ data to send. *timeout* is an integer that specifies the call timeout in millise
 Returns *headers* which is a table of response headers. Returns *body* which is the string response
 body. May be nil if there is no body.
 
+httpCallNonblocking()
```

Comment:
i'm not a huge fan of this name (it sounds a bit clunky/verbose) and it worries me a bit if we are going to end up with one new function per variation of a HTTP call (`httpCallNonblockingWithCallbacks` kinda situation)

are we able to/can we add parameters to existing LUA filters? it'd be nice to either have one or two (e.g. `httpCall` and `asyncHttpCall`) functions with options for whether the response should be dropped, callbacks to invoke on response etc. 

@mattklein123 @jmarantz @dio

Reply:
Are we able to/can we add parameters to existing LUA filters?

I think we can, yeah. This is a reasonable/great ask, yeah we should do this.

Reply:
Yes I think we can do this and SGTM

Reply:
Removed the new function, adding a parameter `asynchronous`.